pkg/idutil/netwalker/netwalker.go

```diff
@@ -1,119 +0,0 @@
-/*
-   Copyright The containerd Authors.
-
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
-
-       http://www.apache.org/licenses/LICENSE-2.0
-
-   Unless required by applicable law or agreed to in writing, software
-   distributed under the License is distributed on an "AS IS" BASIS,
-   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   See the License for the specific language governing permissions and
-   limitations under the License.
-*/
-
-package netwalker
```

Comment:
As discussed in the ticket about `walkers`, this is pattern is introducing complexity with no clear upside.

Furthermore, for the specific case here of network, it is buggy.

Removing entirely.